### PR TITLE
updates to include extra docker step

### PR DESCRIPTION
**Pull request by mirandaauhl**
# Description

Update getting started add data page to include extra docker step. Also update format of table outline, seemed a little off after seeing live?

# Version

Which documentation version does this PR apply to?

- [x] Latest (Default)
- [ ] Version 1.7
- [ ] Older [specify]

### CODE REVIEW

File: timescaledb/getting-started/add-data.md [hunk 1]
```diff
@@ -5,25 +5,23 @@ stock trade data (also known as "tick data") from [twelvedata][twelve-data].
 
 ## About the dataset
 
-The dataset contains second-by second stock-trade data for the top 100 most-traded symbols, in a hypertable named `stocks_real_time`. It also
-includes a separate table of company symbols and company names, in a table named `company`. `company` is not a hypertable.
+The dataset contains second-by second stock-trade data for the top 100 most-traded symbols, in a
```
Comment (charislam) [inline timescaledb/getting-started/add-data.md:8]:
```suggestion
The dataset contains second-by-second stock-trade data for the top 100 most-traded symbols, in a 
```

File: timescaledb/getting-started/add-data.md [hunk 2]
```diff
@@ -40,17 +38,27 @@ includes a separate table of company symbols and company names, in a table named
...
 
-1.  At the command prompt, unzip the `.csv` files:
+1.  In a new terminal window, run this code to unzip the `.csv` files:
     ```bash
     unzip stock_data_real_time.zip
     ```
 
-1.  At the `psql` prompt, insert the data into your 
-    TimescleDB instance. If the `.csv` files aren't in your current directory, replace the file paths as needed:
+1.  For folks using Docker containers, you will have to add the data files to your container before
```
Comment (charislam) [inline timescaledb/getting-started/add-data.md:51]:
```suggestion
1.  If you're using a Docker container, add the data files to your container before 
```

**Review by charislam — APPROVED** [review]

File: timescaledb/getting-started/add-data.md [hunk 3]
```diff
@@ -5,25 +5,23 @@ stock trade data (also known as "tick data") from [twelvedata][twelve-data].
...
 
 ### Table details
-`stocks_real_time`: Stock data. Includes stock price quotes at every second during trading hours.
 
-```bash
+`stocks_real_time`: Stock data. Includes stock price quotes at every second during trading hours.
     * time (timestamptz): timestamp column incrementing second by second
     * symbol (text): symbols representing a company, mapped to company names in the `company` table
     * price (double precision): stock quote price for a company at the given timestamp
     * day_volume (int): number of shares traded each day, NULL values indicate the market is closed
```
Comment (charislam) [inline timescaledb/getting-started/add-data.md:18]:
```suggestion
* time (timestamptz): timestamp column incrementing second by second
* symbol (text): symbols representing a company, mapped to company names in the `company` table
* price (double precision): stock quote price for a company at the given timestamp
* day_volume (int): number of shares traded each day, NULL values indicate the market is closed
```

Assuming this isn't intended to be a code block anymore? One little annoying thing about Markdown: if they're used outside of a nested list, indents automatically create code blocks.

File: timescaledb/getting-started/add-data.md [hunk 4]
```diff
@@ -5,25 +5,23 @@ stock trade data (also known as "tick data") from [twelvedata][twelve-data].
...
-```
 
-`company`: mapping for symbols to company names
 
-```bash
+`company`: mapping for symbols to company names
     * symbol (text): the symbol representing a company name
     * name (text): corresponding company name
```
Comment (charislam) [inline timescaledb/getting-started/add-data.md:23]:
```suggestion
* symbol (text): the symbol representing a company name
* name (text): corresponding company name
```

File: timescaledb/getting-started/add-data.md [hunk 5]
```diff
@@ -40,17 +38,27 @@ includes a separate table of company symbols and company names, in a table named
     );
     ```
 
-1.  Download the following `.zip` file. The file contains two `.csv` files: one with company information, and one with real-time stock trades for one month.
+1.  Download the following `.zip` file. The file contains two `.csv` files: one with company 
+information, and one with real-time stock trades for one month.
```
Comment (charislam) [inline timescaledb/getting-started/add-data.md:42]:
```suggestion
    information, and one with real-time stock trades for one month.
```

**Review by charislam — COMMENTED** [review]
Noticed a bit of broken formatting due to indents

File: timescaledb/getting-started/add-data.md [hunk 6]
```diff
@@ -5,25 +5,23 @@ stock trade data (also known as "tick data") from [twelvedata][twelve-data].
 
 ## About the dataset
 
-The dataset contains second-by second stock-trade data for the top 100 most-traded symbols, in a hypertable named `stocks_real_time`. It also
-includes a separate table of company symbols and company names, in a table named `company`. `company` is not a hypertable.
+The dataset contains second-by second stock-trade data for the top 100 most-traded symbols, in a 
+hypertable named `stocks_real_time`. It also includes a separate table of company symbols and company 
+names, in a table named `company`. `company` is not a hypertable.
```
Comment (Loquacity) [inline timescaledb/getting-started/add-data.md:10]:
```suggestion
names, in an ordinary SQL table named `company`.
```

File: timescaledb/getting-started/add-data.md [hunk 7]
```diff
@@ -40,17 +38,27 @@ includes a separate table of company symbols and company names, in a table named
     );
     ```
 
-1.  Download the following `.zip` file. The file contains two `.csv` files: one with company information, and one with real-time stock trades for one month.
+1.  Download the following `.zip` file. The file contains two `.csv` files: one with company
```
Comment (Loquacity) [inline timescaledb/getting-started/add-data.md:41]:
```suggestion
1.  Download this `.zip` file. The file contains two `.csv` files: one with company 
```

File: timescaledb/getting-started/add-data.md [hunk 8]
```diff
@@ -40,17 +38,27 @@ includes a separate table of company symbols and company names, in a table named
...
+copying them into your database:
+    ```bash
+    docker cp stocks_real_time.csv timescaledb:/stocks_real_time.csv
+
+    docker cp company.csv timescaledb:/company.csv
+    ```
+
+
+1. At the `psql` prompt, insert the data into your TimescleDB instance. If the `.csv` files aren't in 
+your current directory, replace the file paths as needed:
```
Comment (Loquacity) [inline timescaledb/getting-started/add-data.md:61]: The numbering is resetting here:

image

**Review by Loquacity — CHANGES_REQUESTED** [review]

File: timescaledb/getting-started/add-data.md (hunk 6 above)
Comment (mirandaauhl) [inline timescaledb/getting-started/add-data.md:10]:
```suggestion
names, in an ordinary PostgreSQL table named `company`. 
```

File: timescaledb/getting-started/add-data.md [hunk 9]
```diff
@@ -5,25 +5,23 @@ stock trade data (also known as "tick data") from [twelvedata][twelve-data].
...
+hypertable named `stocks_real_time`. It also includes a separate table of company symbols and company 
+names, in an ordinary PostgreSQL table named `company`. 
 
 ### Table details
+
 `stocks_real_time`: Stock data. Includes stock price quotes at every second during trading hours.
+* time (timestamptz): timestamp column incrementing second by second
+* symbol (text): symbols representing a company, mapped to company names in the `company` table
+* price (double precision): stock quote price for a company at the given timestamp
+* day_volume (int): number of shares traded each day, NULL values indicate the market is closed
```
Comment (mirandaauhl) [inline timescaledb/getting-started/add-data.md:18]:
```suggestion
- time (timestamptz): timestamp column incrementing second by second
- symbol (text): symbols representing a company, mapped to company names in the `company` table
- price (double precision): stock quote price for a company at the given timestamp
- day_volume (int): number of shares traded each day, NULL values indicate the market is closed
```

File: timescaledb/getting-started/add-data.md [hunk 10]
```diff
@@ -5,25 +5,23 @@ stock trade data (also known as "tick data") from [twelvedata][twelve-data].
...
-```bash
-    * time (timestamptz): timestamp column incrementing second by second
-    * symbol (text): symbols representing a company, mapped to company names in the `company` table
-    * price (double precision): stock quote price for a company at the given timestamp
-    * day_volume (int): number of shares traded each day, NULL values indicate the market is closed
-```
 
 `company`: mapping for symbols to company names
+* symbol (text): the symbol representing a company name
+* name (text): corresponding company name
```
Comment (mirandaauhl) [inline timescaledb/getting-started/add-data.md:23]:
```suggestion
- symbol (text): the symbol representing a company name
- name (text): corresponding company name
```

File: timescaledb/getting-started/add-data.md (hunk 9 above)
Comment (mirandaauhl) [inline timescaledb/getting-started/add-data.md]:
```suggestion
* **`stocks_real_time`:** Stock data. Includes stock price quotes at every second during trading hours.
   * time (timestamptz): timestamp column incrementing second by second
   * symbol (text): symbols representing a company, mapped to company names in the `company` table
   * price (double precision): stock quote price for a company at the given timestamp
   * day_volume (int): number of shares traded each day, NULL values indicate the market is closed
```

File: timescaledb/getting-started/add-data.md [hunk 11]
```diff
@@ -5,25 +5,23 @@ stock trade data (also known as "tick data") from [twelvedata][twelve-data].
...
+hypertable named `stocks_real_time`. It also includes a separate table of company symbols and company 
+names, in an ordinary PostgreSQL table named `company`. 
 
 ### Table details
+
 `stocks_real_time`: Stock data. Includes stock price quotes at every second during trading hours.
+- time (timestamptz): timestamp column incrementing second by second
+- symbol (text): symbols representing a company, mapped to company names in the `company` table
+- price (double precision): stock quote price for a company at the given timestamp
+- day_volume (int): number of shares traded each day, NULL values indicate the market is closed
```
Comment (mirandaauhl) [inline timescaledb/getting-started/add-data.md:18]:
```suggestion
* **`stocks_real_time`**: Stock data. Includes stock price quotes at every second during trading hours.
   *  time (timestamptz): timestamp column incrementing second by second
   *  symbol (text): symbols representing a company, mapped to company names in the `company` table
   * price (double precision): stock quote price for a company at the given timestamp
   * day_volume (int): number of shares traded each day, NULL values indicate the market is closed
```

**Review by Loquacity — APPROVED** [review]

**Pull request merged**